### PR TITLE
Fix typos and whitespace

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L aci,ake,aline,alle,alledges,alocation,als,ang,anid,anormal,apoints,ba,beginn,behaviour,bloaded,bottome,byteorder,calculater,cancelled,cancelling,cas,cascade,centimetre,childrens,childs,colour,colours,commen,connexion,currenty,dof,doubleclick,dum,eiter,elemente,ende,feld,finde,findf,freez,hist,iff,indicies,initialisation,initialise,initialised,initialises,initialisiert,inout,ist,itsel,kilometre,lod,mantatory,methode,metres,millimetre,modell,nd,noe,normale,normaly,nto,numer,oce,oder,ontop,orgin,orginx,orginy,ot,pard,parm,parms,pres,programm,que,rady,recurrance,ro,rougly,seperator,serie,sinc,siz,strack,substraction,te,technic,thist,thru,tread,uint,unter,uptodate,vertexes,wallthickness,whitespaces -S ./.git,*.po,*.ts,./ChangeLog.txt,./src/3rdParty,./src/Mod/Assembly/App/opendcm,./src/CXX,./src/zipios++,./src/Base/swig*,./src/Mod/Robot/App/kdl_cp,./src/Mod/Import/App/SCL,./src/WindowsInstaller,./src/Doc/FreeCAD.uml,./src/Base/StackWalker.cpp,./build/doc/SourceDocu`

### CODE REVIEW

File: src/Mod/Sketcher/Gui/ConstraintFilters.h [hunk 1]
```diff
@@ -108,7 +108,7 @@ namespace ConstraintFilter {
             return (... | (1 << static_cast<std::underlying_type_t<Args>>(args)));
     }
 
-    /// Array of FilterValue bit sets of size the the number of FilterValues indicating for each FilterValue, which other
+    /// Array of FilterValue bit sets of size of the number of FilterValues indicating for each FilterValue, which other
```
Comment (luzpaz) [inline src/Mod/Sketcher/Gui/ConstraintFilters.h:111]: Is this the correct fix?

File: src/Mod/TemplatePyMod/FeaturePython.py [hunk 2]
```diff
@@ -547,7 +547,7 @@ def __init__(self, obj):
 		sep2.addChild(coin.SoSphere())
 		obj.RootNode.addChild(sep1)
 		obj.RootNode.addChild(sep2)
-		# triggers an updateData call so the the assignment at the end
+		# triggers an updateData call so the assignment at the end
```
Comment (luzpaz) [inline src/Mod/TemplatePyMod/FeaturePython.py:550]: Grammar is confusing here. Please advise

File: src/Mod/TechDraw/Templates/zh_CN/landscape/A0_Landscape_CN_CLIP.svg [hunk 3]
```diff
@@ -275,7 +275,7 @@ e">
         <polygon points="598,828 606,828 602,834" />
 	</g>
 
-	<g id="bunding hole">
+	<g id="binding hole">
```
Comment (luzpaz) [inline src/Mod/TechDraw/Templates/zh_CN/landscape/A0_Landscape_CN_CLIP.svg:278]: Click the `<>` button in the top right of this window to view the SVG code. Here is the fix I made.

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7380">pipeline status</a> for feature branch [PR_7380](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7380). Pipeline [617050747](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/617050747) was triggered at [79a6cb3](https://github.com/FreeCAD/FreeCAD/pull/7380/commits/79a6cb3a4a4b1ec643df82ed3b4e0a94986196e0). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Can someone kindly review this ?

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7380">pipeline status</a> for feature branch [PR_7380](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7380). Pipeline [618715539](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/618715539) was triggered at [d17936f](https://github.com/FreeCAD/FreeCAD/pull/7380/commits/d17936f3ff69848e1f5e25e1973960050fe22c7d). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by luzpaz** [comment]
Looks like the CI is failing due to an unrelated TechDraw test.

File: src/Mod/TechDraw/Templates/zh_CN/landscape/A0_Landscape_CN_CLIP.svg (hunk 3 above)
Comment (luzpaz) [reply src/Mod/TechDraw/Templates/zh_CN/landscape/A0_Landscape_CN_CLIP.svg:278]: @WandererFan can you confirm this fix is kosher?

**Comment by WandererFan** [comment]
group ids are not used for anything.  you can change it to anything.

**Pull request merged**